### ISSUE TITLE
Error when closing 

### ISSUE THREAD

**Issue by Amixx311** [swiftyapp/swifty#128]
```
TypeError: this.window[t.message] is not a function
    at window:message (C:\Users\Asus\AppData\Local\Programs\Swifty\resources\app.asar\main\index.js:2:7967168)
    at IpcMainImpl.<anonymous> (C:\Users\Asus\AppData\Local\Programs\Swifty\resources\app.asar\main\index.js:2:7970843)
    at IpcMainImpl.emit (node:events:527:28)
    at EventEmitter.<anonymous> (node:electron/js2c/browser_init:165:11237)
    at EventEmitter.emit (node:events:527:28)
```

---

Swifty 0.6.10
Electron 21.2.2
win32 10.0.19044
Locale: en-US

**Comment by Phil353556** [comment]
Hi,
Seems like https://github.com/swiftyapp/swifty/issues/122, except the version of Electron is higher in your case.
Phil

**Comment by alchaplinsky** [comment]
This should be fixed in [v0.6.11](https://github.com/swiftyapp/swifty/releases/tag/v0.6.11)
